weave/Scripts/Player.cs

```diff
@@ -1,19 +1,35 @@
 using Godot;
+using GodotSharper.AutoGetNode;
+using GodotSharper.Instancing;
+using weave.Utils;
 
+namespace weave;
+
+[Instantiable(ObjectResources.PlayerScene)]
 public partial class Player : RigidBody2D
 {
     [Signal]
     public delegate void PlayerShotBulletEventHandler(Node2D bullet, Vector2 globalPosition);
 
-    private const int MovementSpeed = 100;
-    private PackedScene _bombScene = GD.Load<PackedScene>("res://Objects/Bomb.tscn");
+    private const int MovementSpeed = 500;
+
+    [GetNode("Label")]
+    private Label _label;
+
+    public string PlayerId
```

Comment:
Again, maybe not a good idea to return the label value when getting/setting the player ID